### PR TITLE
allow user to set  Admin Server Name at runtime 

### PR DESCRIPTION
**Pull request by mriccell**
Remove WLS 12.1.3 dockerfiles, thi8s is an unsupported version of WebLogic Sertver
Fix https://github.com/oracle/docker-images/issues/2364 allow user to set  Admin Server Name at runtime

### CODE REVIEW

File: OracleWebLogic/dockerfiles/12.2.1.3/container-scripts/createAndStartEmptyDomain.sh [hunk 1]
```diff
@@ -60,17 +56,20 @@ if [ $ADD_DOMAIN -eq 0 ]; then
...
+   echo "username=${USER}" >> $DOMAIN_HOME/servers/${ADMIN_NAME}/security/boot.properties
+   echo "password=${PASS}" >> $DOMAIN_HOME/servers/${ADMIN_NAME}/security/boot.properties
    ${DOMAIN_HOME}/bin/setDomainEnv.sh
 fi
 
 # Start Admin Server and tail the logs
 ${DOMAIN_HOME}/startWebLogic.sh
-touch ${DOMAIN_HOME}/servers/AdminServer/logs/AdminServer.log
-tail -f ${DOMAIN_HOME}/servers/AdminServer/logs/AdminServer.log &
+if [  -e ${DOMAIN_HOME}/servers/${ADMIN_NAME}/logs/${ADMIN_NAME}.log ]; then
```
Comment (rjeberhard) [inline OracleWebLogic/dockerfiles/12.2.1.3/container-scripts/createAndStartEmptyDomain.sh:68]: You have two spaces ahead of "-e". You only need one.

File: OracleWebLogic/dockerfiles/12.2.1.3/README.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ You can override the default values of the following parameters during runtime w
 
 **NOTE**: For security, the Administration port 9002 is enabled by default, before running the container in WebLogic 12.2.1.3 the patch 27117282 must be applied. Please download the patch and apply it after you have built the 12.2.1.3 image. You can follow the sample https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-patch to see how to patch. An alternative is to not enable Administration port when you issue the docker run command, set `ADMINISTRTATION_PORT_ENABLED` to false. If you intend to run these images in production, then you must change the Production Mode to `production`. When you set the `DOMAIN_NAME`, the `DOMAIN_HOME=/u01/oracle/user_projects/domains/$DOMAIN_NAME`.
 
-	$ docker run -d -p 7001:7001 -p 9002:9002  -v `HOST PATH where the domain.properties file is`:/u01/oracle/properties -e ADMINISTRATION_PORT_ENABLED=true -e DOMAIN_NAME=docker_domain oracle/weblogic:12.2.1.3-developer
+	$ docker run -d -p 7001:7001 -p 9002:9002  -v `HOST PATH where the domain.properties file is`:/u01/oracle/properties -e ADMINISTRATION_PORT_ENABLED=true -e DOMAIN_NAME=docker_domain -e ADMIN_NAME=docker-AdminServer oracle/weblogic:12.2.1.3-developer
```
Comment (rjeberhard) [inline OracleWebLogic/dockerfiles/12.2.1.3/README.md:75]: Does this mean that we no longer have a default for ADMIN_NAME or just that it is now fully configurable? I would prefer if pre-existing `docker run` commands continue to work.

File: OracleWebLogic/dockerfiles/14.1.1.0/container-scripts/createAndStartEmptyDomain.sh [hunk 3]
```diff
@@ -59,17 +56,20 @@ if [ $ADD_DOMAIN -eq 0 ]; then
 
    # Create an empty domain
    wlst.sh -skipWLSModuleScanning -loadProperties $PROPERTIES_FILE  /u01/oracle/create-wls-domain.py
-   mkdir -p ${DOMAIN_HOME}/servers/AdminServer/security/
-   chmod -R  g+w  ${DOMAIN_HOME}
-   echo "username=${USER}" >> $DOMAIN_HOME/servers/AdminServer/security/boot.properties
-   echo "password=${PASS}" >> $DOMAIN_HOME/servers/AdminServer/security/boot.properties
+   mkdir -p  ${DOMAIN_HOME}/servers/${ADMIN_NAME}/security/
```
Comment (rjeberhard) [inline OracleWebLogic/dockerfiles/14.1.1.0/container-scripts/createAndStartEmptyDomain.sh:59]: Unnecessary space added here too (after "-p")

File: OracleWebLogic/dockerfiles/14.1.1.0/container-scripts/createAndStartEmptyDomain.sh [hunk 4]
```diff
@@ -59,17 +56,20 @@ if [ $ADD_DOMAIN -eq 0 ]; then
...
+   chmod -R g+w  ${DOMAIN_HOME}
```
Comment (rjeberhard) [inline OracleWebLogic/dockerfiles/14.1.1.0/container-scripts/createAndStartEmptyDomain.sh:60]: While we're at it... Extra space here too (after "g+w")

File: OracleWebLogic/dockerfiles/12.2.1.3/README.md (hunk 2 above)
Comment (mriccell) [reply OracleWebLogic/dockerfiles/12.2.1.3/README.md:75]: No there is a default value in Dockerfile as well as in the README

**Review by rjeberhard — APPROVED** [review]

**Pull request merged**